### ISSUE TITLE
Peer review for hw09-Sukeysun

### ISSUE THREAD

**Issue by ziqiangt** [STAT545-UBC-students/hw09-Sukeysun#2]
|    **Topic**    | **Excellent** |**Satisfactory** |**Needs Work** |
|----------------|---------------|------------------|------------------|
| Coding style  |     :thumbsup:   |                     |                           |
| Coding strategy      |   :thumbsup:  |                         |                           |
| Presentation: graphs |  |           :thumbsup:               |                           |
| Presentation: tables | :thumbsup:   |                        |                          |
| Achievement, activity |   :thumbsup:  |                        |                           |
| Ease of access |  :thumbsup:    |                    |                       |

Remarks:
Sukeysun has answered the basic questions and has gone beyond what was expected and required. Here are some comments that might interest you. 
1. Good readme file. It is very clear to know what you did.
2. Good Makefile visualization. You did good job to use python. Wow!
3. It would be much better if you could give a visualization html link.
4. I tried to view your report.html file. it seems there is no graphs. 

 In conclusion, it is a good sample answer and I learned a lot from you.
